backend/accounts.go

```diff
@@ -58,6 +58,10 @@ const accountsHardLimit = 5
 // AccountsList is an accounts.Interface slice which implements a lookup method.
 type AccountsList []accounts.Interface
 
+// KeystoresAccountsListMap is a map where keys are keystores' fingerprints and values are
+// AccountsLists of accounts belonging to each keystore.
+type KeystoresAccountsListMap map[*config.Keystore]AccountsList
```

Comment:
why is this in accounts.go but `AccountsByKeystore()` which uses it is in backend.go? seems like they should both be in accounts.go.

Reply:
Addressed in #2609